classes/battle/BattleEffectsManager.php

```diff
@@ -423,7 +428,7 @@ public function applyActiveEffect(Fighter $target, Fighter $attacker, BattleEffe
             return false;
         }
 
-        if($effect->effect == 'residual_damage' || $effect->effect == 'bleed') {
+        if($effect->effect == 'residual_damage' || $effect->effect == 'bleed' || $effect->effect == 'delayed_residual' || $effect->effect == 'reflect_damage') {
```

Comment:
This looks like it might be time to make a "damage over time effects" constant array and check here

Reply:
Makes sense especially since we use this multiple times.